ts_export/src/error.rs

```diff
@@ -21,7 +21,7 @@ pub enum TsExportError {
     #[error("Could not resolve type {:?}", _0)]
     UnsolvedType(syn::Type),
     #[error("Could not resolve field {:?}", _0)]
-    UnsolvedField(syn::Field),
+    UnsolvedField(String),
```

Comment:
`syn::Field` gives `Ident` + `Type` information. `String` has no semantic meaning, this could either contain the name of the field, or the name of the type. In any case, we lose information that I dont particularly want to lose since this will surely be useful for error messages down the road.